mbed-hal/pinmap.h

```diff
@@ -1,5 +1,5 @@
 /* mbed Microcontroller Library
- * Copyright (c) 2006-2013 ARM Limited
+ * Copyright (c) 2006-2015 ARM Limited
```

Comment:
Whenever you do this, it makes sense to also add a reference to the SPDX identifier. An example: https://github.com/ARMmbed/uvisor-helloworld/blob/master/source/box-challenge-hw.h#L3